### ISSUE TITLE
Player Indicators - Add Bot Busting Option

### ISSUE THREAD

**Issue by Estascape** [runelite/runelite#12245]
It would be cool to see an option added to the Player Indicators plugin which would remove the highlighted player's name once you've reported them and added to ignore. The plugin could search your ignore list and remove highlighted usernames from players near you. This would be useful for heavily-botted areas where there are stacks of bots on top of each other. It would help the playerbase report more accurately and assure us that we've gotten all of them!

**Comment by MarbleTurtle** [comment]
Would go against the guidelines that Jagex provided as would fall under "Reorders or removes player-based options, such as ‘Trade with’" in this case removing the player entirely

**Comment by Estascape** [comment]
> Would go against the guidelines that Jagex provided as would fall under "Reorders or removes player-based options, such as ‘Trade with’" in this case removing the player entirely

It wouldn’t remove the player entirely from your screen, it would only remove their username from above their head when using the Player Indicator Plugin. This would let you know that you’ve reported them already.

If that’s still breaking ToS, it’s understood. Just thought it’d be a cool little extra.

**Comment by pilino1234** [comment]
Should this only remove the player name on the game screen, or also from the minimap and highlighted tiles if those features are enabled? I think hiding all possible highlights for ignored players would be most consistent.

**Comment by Estascape** [comment]
> 
> 
> Should this only remove the player name on the game screen, or also from the minimap and highlighted tiles if those features are enabled? I think hiding all possible highlights for ignored players would be most consistent.

I don't really pay attention to the minimap, but this sounds great. From the surface it doesn't seem like it would be breaking ToS. The entire purpose is just to hide the Username of the player that has been reported and added to ignore. Their character will still remain on your screen, they just won't have a username above their head anymore.

It'll make reporting way more accurate and less of a hassle when there are loads of bots on top of each other or in the same area. As you continue to report, less and less names will be visible since they are being removed as you add them to your ignore list.

Of course this can be toggled on and off if you're someone who PK's and needs to see usernames while you're in the wild.

**Comment by pilino1234** [comment]
> From the surface it doesn't seem like it would be breaking ToS. The entire purpose is just to hide the Username of the player that has been reported and added to ignore. Their character will still remain on your screen, they just won't have a username above their head anymore.

Correct, the whole ToS thing was most likely a misunderstanding. The player indicators are entirely RuneLite's, so we can decide ourselves when or how we want them to appear.

**Comment by Estascape** [comment]
> Correct, the whole ToS thing was most likely a misunderstanding. The player indicators are entirely RuneLite's, so we can decide ourselves when or how we want them to appear.

It doesn't necessarily have to be an option within Player Indicators; it could be a separate plugin in the Plugin Hub.
However, it might be easier to just code it in since the code is already there.

Not a big deal, just an idea I had for some time now and thought I'd share.
Appreciate your guys' time :)
